app/sql.js

```diff
@@ -385,6 +387,7 @@ async function updateToSchemaVersion4(currentVersion, instance) {
       type STRING,
       members TEXT,
       name TEXT,
+      swarmNodes TEXT,
```

Comment:
Agree with sacha here, since we don't use `swarmNodes` to make a query, it's alright to just leave it in the json stuff and not make a db column for it.